src/client/components/dashboard/dashboard_robot/model.ts

```diff
@@ -108,12 +108,17 @@ export class DashboardRobotModel {
   })
 
   @computed
+  public get connected(): boolean {
+    return this.robot.connected
+  }
+
+  @computed
   public get name(): string {
     return this.robot.name
   }
 
   @computed
-  public get visible(): boolean {
```

Comment:
Moved the "visibility" to be a view model concern, since it now depends where it is shown.